htdocs/core/lib/functions.lib.php

```diff
@@ -234,6 +234,25 @@ function isModEnabled($module)
 	//return !empty($conf->$module->enabled);
 }
 
+/**
+ * isDolValidTms check if a timestamp is valid.
+ *
+ * @param  int|string|null $timestamp timestamp to check
+ * @return bool
+ */
+function isDolValidTms($timestamp)
+{
+	if ($timestamp == '') {
```

Comment:
May be 
`if ($timestamp === '') {`
here ?

Reply:
I also use a shorter nam